eclipse/tests/src/nl/tudelft/watchdog/logic/ui/WatchDogEventManagerTest.java

```diff
@@ -39,10 +40,13 @@
  * eventually transfered to the server, this is one of the most crucial parts of
  * WatchDog. Tests could flicker because they deal with timers (and Java gives
  * no guarantee as to when these timers will be executed).
+ *
+ * TODO(timvdlippe): This test is disabled until we can properly deflake it.
```

Comment:
Move this to right above the @Ignore?

Reply:
That would break the JavaDoc generation. I have moved the `@Ignore` to the first line